pkg/cmd/plugincheck2/quickmeta.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// GetIDAndVersion Gets plugin id and version from extracted zip
+func GetIDAndVersion(archiveDir string) (*string, *string, error) {
```

Comment:
Why are we returning references to strings instead of values?

Reply:
doesn't have to, was just returning nil/nil/error vs "","",err

Reply:
updated this also